crypto/evp/s_lib.c

```diff
@@ -294,12 +285,11 @@ EVP_SKEY *EVP_SKEY_to_provider(EVP_SKEY *skey, OSSL_LIB_CTX *libctx,
     if (ctx.keydata == NULL)
         goto err;
 
-    ret = evp_skey_alloc();
+    ret = evp_skey_alloc(skeymgmt);
```

Comment:
Lines 261 and 266 fetch a 'skeymgmt'. If this returns NULL then line 271 is going to crash.

Reply:
Fixed.